composer.json

```diff
@@ -16,7 +16,7 @@
 		"wmde/email-address": "~1.0",
 		"wmde/euro": "~1.0",
 		"wmde/clock": "~1.0",
-		"wmde/fundraising-donations": "~16.0",
+		"wmde/fundraising-donations": "dev-donortype-enum",
```

Comment:
needs new release of `fundraising-donations` when [the `fundraising-donations](https://github.com/wmde/fundraising-donations/pull/233)  is merged